### PR TITLE
M1 day 3 dev task theja

### PR DESCRIPTION
**Pull request by Thejuhoney**
updated day 1 and day 2 notes in day 3 task ,and interview questions in cheatsheet

### CODE REVIEW

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]
Please move the changes to cheatsheet

**Review by Thejuhoney — COMMENTED** [review]
day 1 and 2 presentation notes updated in task file and interview questions updated in cheetsheet

**Comment by IshaaqhShaik** [comment]
Please resolve the merge conflicts and clean the commit history as well there are multiple commits from multiple developers.

**Comment by Kunimohan** [comment]
> Please resolve the merge conflicts and clean the commit history as well there are multiple commits from multiple developers


resolved merge conflicts and updated commits

File: Module-1/Day-3/Cheatsheet/cheatsheet.txt [hunk 1]
```diff
@@ -52,4 +52,21 @@
   hotfix
 
 10.How to create branch?
-    git checkout -b branchname
\ No newline at end of file
+    git checkout -b branchname
+
+
+Q1 what is git
```
Comment (IshaaqhShaik) [inline Module-1/Day-3/Cheatsheet/cheatsheet.txt:58]: Please indent and name the questions accordingly

**Review by IshaaqhShaik — CHANGES_REQUESTED** [review]

**Review by IshaaqhShaik — APPROVED** [review]

**Pull request merged**